rosa_release_notes/rosa-release-notes.adoc

```diff
@@ -16,6 +16,9 @@ toc::[]
 [id="rosa-q2-2024_{context}"]
 === Q2 2024
 
+* **Approved Access for ROSA clusters.** Red{nbsp}Hat Site Reliability Engineering (SRE) managing and proactively supporting ROSA Clusters will typically not require access to customer Data as part of the normal operations. In the unlikely event should Red{nbsp}Hat SRE (Site Reliability Engineer) need access to customer data, the *Approved Access* functionality provides an interface for customers to review and *approve* or *deny* access requests.
```

Comment:
I forgot to submit this comment in the earlier review, but same comments I made in the other files could be applied to these as well.

This is also a pretty large block of text for users to read, so I'd recommend trying to break it up some. Even putting a "+" between these two paragraphs to make them separate paragraphs could help